src/documents/ModelRelay.md

```diff
@@ -4,43 +4,42 @@ layout: default
 category: Infusion
 ---
 
-The Infusion Model Relay system is a powerful scheme for supplying declarative configuration that specifies rules for keeping multiple pieces of model state around the component tree automatically up to date with each other's changes. 
+Using the Infusion Model Relay system, you can supply declarative configuration which specifies rules for keeping multiple pieces of model state around the component tree automatically up to date with each other's changes. 
 
-We sometimes use the term _**model skeleton**_ to refer to a connected set of models around the component tree which are related together by relay rules. This appeals to the idea that these models form an inner core which
+We sometimes use the term _**model skeleton**_ to refer to a connected set of models around the component tree which are related together by relay rules. This illustrates the idea that these models form an inner core which
```

Comment:
Arrive is in theory correct, since it is the models which arrive - especially since the next verb comes with "before they start". However the sentence is unclear and I've tried to improve it a bit. Still not great. If only we had a _technical writer_.